src/plugins/codeStyle/index.spec.ts

```diff
@@ -828,6 +828,264 @@ describe('codeStyle', () => {
             expect(actualSrc).to.equal(expectedSrc);
         });
 
+        it('BRS file color format is quoted-numeric-hex and case is uppercase', async () => {
```

Comment:
Tests have been added inside the `describe('fix', () => {` block.
You should add a new `describe('colors'` block (before the fixes) for those tests.